tests/Feature/WebRouteTest.php

```diff
@@ -5,33 +5,83 @@
 use Illuminate\Foundation\Testing\RefreshDatabase;
 use Illuminate\Foundation\Testing\WithFaker;
 use Tests\TestCase;
+use Inertia\Testing\Assert;
+use App\Models\User;
 
 class WebRouteTest extends TestCase
 {
     use RefreshDatabase;
-    
+
     /**
-     * Test the /auth/api_settings route
+     * Test the / route
      *
      *  @return void
      */
-    public function test_welcome_route()
+    public function test_home_route()
     {
         $response = $this->get('/');
 
-        $response->assertStatus(302);
+        $response->assertSuccessful();
+        $response->assertViewIs('app')
+            ->assertInertia(function (Assert $page) {
+                $page->component('Home');
+            });
+    }
+
+     /**
+     * Test the / route when authenticated
+     *
+     *  @return void
+     */
+    public function test_home_page_has_user()
+    {
+        $user = User::factory()->create();
+
+        $response = $this->actingAs($user)->get('/');
+
+        $response->assertSuccessful();
+        $response->assertViewIs('app')
+            ->assertInertia(function (Assert $page) use ($user) {
+                $page->component('Home')
+                    ->where('user.name', $user->username);
+            });
+    }
+
+    /**
+     * Test the /store route
+     *
+     *  @return void
+     */
+    public function test_store_route()
+    {
+        $response = $this->get('/store');
+
+        $response->assertRedirect(route('store.api-settings'));
+    }
+
+    /**
+     * Test the authenticated /store/api-settings route
```

Comment:
Hm, actually, this test now covers both, authenticated and non-authenticated access to that route. Does that mean browser tests are needed to actually test the difference between the two?

PS: Found it.